internal/receiver/simpleprometheusremotewritereceiver/config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simpleprometheusremotewritereceiver
+
+import (
+	"errors"
+	"time"
+
+	"github.com/jaegertracing/jaeger/pkg/multierror"
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/config/confignet"
+)
+
+var _ component.Config = (*Config)(nil)
+
+const (
+	typeString = "simpleprometheusremotewrite"
+)
+
+type Config struct {
+	ListenAddr    confignet.NetAddr `mapstructure:",squash"`
```

Comment:
Would embedding https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/confighttp/confighttp.go#L206 make more sense? not clear on why transport would be configurable.

Reply:
statsd and carbon both support udp where this is always http tmk. http2 option seems nice to have, but not necessarily needed immediately.